src/cms/content-menu.md

```diff
@@ -37,6 +37,10 @@ Manage the [visual presentation]({% link design/design-theme.md %}) of your stor
 
 - Use the [schedule]({% link design/schedule.md %}) to plan theme changes in advance for a season or promotion.
 
+{:.bs-callout-info}
+When the new **Media Gallery** is enabled by setting the **Enable Enhanced Media Gallery** option to `Yes` under the **Stores** > _Settings_ > **Configuration** > **Advanced** > **System** menu, a new tab called `Media` appears with a single option to access the `Media Gallery`.
+![Media Gallery Menu]({% link images/images/enhanced-media-gallery/media_gallery.png %}){: .zoom}
```

Comment:
Needs a blank line above

```suggestion

![Media Gallery Menu]({% link images/images/enhanced-media-gallery/media_gallery.png %}){: .zoom}
```

Reply:
If I remove this space, then the image associated with the warning message goes out and looks different

Reply:
Are you trying to add the image to the Info box?

Reply:
@dineshvb I will play with this, but I think we should probably replace the menu image at the top to include this and then put the note below that so they know it requires a config setting to be enabled. Right now it is part of the Design section on the page, which is a whole different set of functionality. 

Also, I just did new 2.4 test builds and the config setting still appears as "Enable Old Media Gallery" -- is the screen you added in the PR description part of an update in a future release?

Reply:
> Are you trying to add the image to the Info box?

Yes

Reply:
> @dineshvb I will play with this, but I think we should probably replace the menu image at the top to include this and then put the note below that so they know it requires a config setting to be enabled. Right now it is part of the Design section on the page, which is a whole different set of functionality.
> 
> Also, I just did new 2.4 test builds and the config setting still appears as "Enable Old Media Gallery" -- is the screen you added in the PR description part of an update in a future release? Or maybe it is old? I know they decided not to call in "Enhanced Media Gallery" at one point ([#507 (comment)](https://github.com/magento/merchdocs/pull/507#issuecomment-660202604)).

I am working on a 2.4 instance. I am getting the Enhanced Media Gallery menu. I think it is old

Reply:
We try to avoid putting large images the info boxes, plus having the a screen of the menu two different ways in the page is kind of overkill. I think my adjustment to replace the one ot the top and add the note underneath provides what we need on that issue.

If your version is older than the 2.4.2 release, then the path to the configuration and the name of the option should change to it was documented.

![image](https://user-images.githubusercontent.com/1128913/105763977-fdb22b80-5f1b-11eb-9622-9213272a3472.png)

Reply:
Should I move the image above and add a note there?

Reply:
I already did that. I will update the note message according to the current labeling of the config option. Then we should be in good shape to test and merge.